ui/client/components/AddProcessDialog.js

```diff
@@ -105,5 +118,21 @@ function mapState(state) {
   }
 }
 
+const nameAlreadyExists = (processes, subprocesses, name) => {
+  return processes.map(process => process.name).includes(name)
+    || subprocesses.map(subProcess => subProcess.name).includes(name)
+}
+
+const validators = [
+  {
+    isValid: (processes, subprocesses, name) => notEmptyValidator.isValid(name),
+    message: notEmptyValidator.message
+  },
+  {
+    isValid: (processes, subprocesses, name) => !nameAlreadyExists(processes, subprocesses, name),
+    message: duplicate("Id")
```

Comment:
Validations are always shown in the context of a field. If a field is the "Id" we do not need to repeat this information.

Reply:
I've changed it to "This value is already taken"